### DISCUSSION TITLE
detect notification banner being closed?

### DISCUSSION THREAD

**Discussion by 321github123** [RocketCommunicationsInc/astro#118] [Q&A] [answered]
Hi,

I am interested in using the React component "RuxNotification".  Is there anyway to detect if the user has closed the banner or not?

Thank you.

Mark Lai

**Comment by markacianfrani** [comment, marked as answer]
> Hi,
> 
> 
> 
> I am interested in using the React component "RuxNotification".  Is there anyway to detect if the user has closed the banner or not?
> 
> 
> 
> Thank you.
> 
> 
> 
> Mark Lai

Not currently but we'll add an event you can listen to ASAP! Also working on a hotfix now to fix an issue with the close icon not closing on click in case you were running into that as well.

**Reply by markacianfrani** [reply]
We've just released 6.0.3 which fixes an issue with the close icon and also exposes a new `ruxclosed` event so that you can detect when the user closes the banner

**Reply by 321github123** [reply]
Wonderful! Thanks!
